### ISSUE TITLE
Cant' access terminal

### ISSUE THREAD

**Issue by tavareslfelipe** [recalbox/recalbox-os#1115]
### Before posting issue 
- For support go to **IRC** or **forum**. Github is only for development issue/improvement 
- Be sure your issue is not already reported : check https://github.com/recalbox/recalbox-os/wiki/Known-Bugs-%28EN%29
- Format your title's issue as follow : 
- [Full Version][optional : KODI|ES|Buidroot|etc] short desc 

ex: [4.0.0-beta1][ES] keyboard dosbox doesn't work 

---
Please fill the following form :

### What happened
I'm trying to access Recalbox terminal, but it isn't working. I'm following this tutorial (https://github.com/recalbox/recalbox-os/wiki/Root-access-on-terminal-%28EN%29), but when i press "F4" or "Alt+F2", nothing happens and i can't exit from the emulationstation's screen. I'm using a wired Mac keyboard.  
### What should have happened
Access terminal.
### Enumerate your devices
Raspberry Pi 3+
Recalbox OS4
### How to reproduce the bug

**Comment by substring** [comment]
F4 quits ES
Alt+F2 switches to tty2. I guess your mac keyboard is troublesome ;)

**Comment by coquillo** [comment]
I have the same problem , I can´t access to the terminal , I try with "F4", or Alt+F2 .. doesn´t work...

:(

**Comment by uabassguy** [comment]
I also cant access tty2, using a Rii mini i8 wireless keyboard

**Comment by substring** [comment]
@coquillo it's not "try this or try that", you must respect the sequence : F4, wait that ES shutdowns, then ALT+F2. If either key sequence doesn't work, then your keyboard may not work with recalbox

@uabassguy does pressing F4 quits ES at least ?

**Comment by uabassguy** [comment]
It isnt. It may be due to another issue I'm having where the ES UI appears over any game I try (updated from the previous version just now), I'll file that separately.

**Comment by substring** [comment]
@uabassguy I'd say you have 2 Es instances running. Maybe you followed the tutorial to use a NAS for roms and forgot that upon 4.0 update ?

**Comment by uabassguy** [comment]
That's correct. Should I reapply that?

**Comment by substring** [comment]
You must have 2 emulationstation init scripts. AFAIR the tutorial says you should remove the S31 one

**Comment by uabassguy** [comment]
You're right about the 2 init scripts. I did rm S92emulationstation from /etc/init.d and the roms disappeared though. So I added mount -a after start) and did mv S31 to S91 (after smb) and it all seems to be working fine again. Thank you! Sorry to bother you guys at 2:30 AM

**Comment by substring** [comment]
we are on CET here :) so it's late morning :)

**Comment by LipkeGu** [comment]
We could implement an easy Fix?: 
Add a menu option to exit ES2 which calls ```/etc/init.d/S31emulationstation stop```

**Comment by rockaddicted** [comment]
-> press F4 button
